internal/pkg/manifest/env.go

```diff
@@ -229,7 +273,17 @@ type publicHTTPConfig struct {
 	Certificates []string `yaml:"certificates,omitempty"`
 }
 
+// IsEmpty returns true if there are customization to the public ALB.
+func (cfg publicHTTPConfig) IsEmpty() bool {
+	return len(cfg.Certificates) == 0
+}
+
 type privateHTTPConfig struct {
 	InternalALBSubnets []string `yaml:"subnets,omitempty"`
 	Certificates       []string `yaml:"certificates,omitempty"`
 }
+
+// IsEmpty returns true if there are customization to the internal ALB.
```

Comment:
I could be confused, but did you mean this?
```suggestion
// IsEmpty returns true if there is no customization to the internal ALB.
```